portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,43 +42,42 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String defaultCommentLimit = "5";
-    int commentLimit = Integer.parseInt(getParameter(request, "numShown", defaultCommentLimit));
+    int commentLimit = getIntInpOrDefault(request, "commentLimit", defaultCommentLimit);
 
     String defaultPageNumber = "0";
-    int pageNumber = Integer.parseInt(getParameter(request, "pageNumber", defaultPageNumber));
+    int pageNumber = getIntInpOrDefault(request, "pageNumber", defaultPageNumber);
 
-    String defaultSortMethod = "DESCENDING";
-    String sortMethod = getParameter(request, "sortMethod", defaultSortMethod);
+    String defaultSortMethod = "true";
```

Comment:
"true" isn't a sort method.  Also this doesn't really need to be a variable.

Reply:
Did some renaming (method -> direction); not totally perfect but i think it is more clear without being overly verbose